packages/react/src/components/form/Checkbox/Checkbox.mdx

```diff
@@ -10,7 +10,7 @@ import * as CheckboxGroupStories from './Group/Group.stories';
 
 # Checkbox
 
-Vi bruker `Checkbox` (avmerkingsboks) for å gi brukerne valg, der de kan velge ett eller flere alternativer. Brukerne kan både velge og oppheve valgene de gjør. Bruk [`Checkbox.Group`](/docs/komponenter-checkbox-group--docs) til å gruppere flere valg.
+Vi bruker `Checkbox` (avmerkingsboks) for å gi brukerne valg, der de kan velge ett eller flere alternativer. Brukerne kan både velge og oppheve valgene de gjør. Bruk [`Checkbox.Group`](/story/komponenter-checkbox-group--preview) til å gruppere flere valg.
```

Comment:
Ah, det ser ut som en glipp ja! 

Den er embeded i samme docs/mdx fil (linje 47) så kanskje en deeplink blir mer riktig? 
```suggestion
Vi bruker `Checkbox` (avmerkingsboks) for å gi brukerne valg, der de kan velge ett eller flere alternativer. Brukerne kan både velge og oppheve valgene de gjør. Bruk [`Checkbox.Group`](/docs/komponenter-checkbox--docs#checkboxgroup) til å gruppere flere valg.
```

Reply:
hmm, ser ikke ut som Storybook docs har støtte for scrollTo 😐

Reply:
Eg trur kanskje ikkje me trenger å linke vidare her, den ligger på samme side.
Ellers, så kan me brude `id`en på headingen, men dette reloader sida
```mdx
[`Checkbox.Group`](#checkboxgroup)
```